apache/yunikorn-site
# [YUNIKORN-2595] use cdn for source download

### What is this PR for?
Add closer.lua to first src link, remove the 1.2.0 release
Fix IDE warnings on download page.

### What type of PR is it?

### What is the Jira issue?
* https://issues.apache.org/jira/browse/YUNIKORN-2595

### How should this be tested?
The current release download page should point to the cdn as per the ASF requirements, not directly open a "save as" for the download. The page that is opened should look like the screenshot below.

### Screenshots (if appropriate)
Correct download page
![closer_lua_page](https://github.com/apache/yunikorn-site/assets/7152971/34780756-0bc6-4ca0-961f-8532879acd75)
